internal/terminal/list_test.go

```diff
@@ -0,0 +1,73 @@
+package terminal
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestList(t *testing.T) {
+	assert.RegisterOpts(reflect.TypeOf(list{}), cmp.AllowUnexported(list{}))
+	message := "a list message"
```

Comment:
`message` is shadowed here later on in the `list.Message()` test.  in light of that, lets shift these tests around a bit:

```go
func TestList(t *testing.T) {
  assert.RegisterOpts...
  t.Run("Should create a new list with its data parsed", func(t *testing.T) {
    list := newList(
      "a list message",
      []interface{}{ ... },
    )

    expectedList := list{ ... }
    assert.Equal(t, expectedList, list)

    t.Run("And Message should ...", func(t *testing.T){ /* can use the list var directly here */ })

    t.Run("And Payload should ...", func(t *testing.T){ /* can use the list var directly here */ })
  })
}
```

of course, you cold simply just rename this var to `listMessage` or something unique too, but it doesn't seem we use/reference this much in the first place, just mostly for creating the initial test list.  let me know what you think!